app/src/main/scala/org/alephium/app/ApiConfig.scala

```diff
@@ -81,7 +82,8 @@ object ApiConfig extends StrictLogging {
         as[Duration]("askTimeout"),
         apiKeys,
         as[U256]("gasFeeCap"),
-        as[Int]("defaultUtxosLimit")
+        as[Int]("defaultUtxosLimit"),
+        as[Int]("maxFormBufferedKiloByte")
```

Comment:
I'd go for `Bytes` instead of `KiloByte`, and use `128 * 1024` in the conf file.